docs/pages/docs/api-reference/generative-ui/render.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: render
+---
+
+import { Callout, Tabs, Tab, Steps } from 'nextra-theme-docs';
+import { UIPreviewCard, Card } from '@/components/home/card';
+import { Browser } from '@/components/home/browser';
+import { EventPlanning } from '@/components/home/event-planning';
+import { Searching } from '@/components/home/searching';
+
+# render()
+
+The `render` function is a helper function to create a streamable UI from an LLM response. It currently only supports
+OpenAI API compatible models with function calling, like gpt-3.5-turbo and [FireFunction](https://fireworks.ai/blog/firefunction-v1-gpt-4-level-function-calling).
+`render` provides a generator-based API for streaming updating UIs from the LLM.
```

Comment:
If fact, the “render” function can be one of the four:
- () => ReactNode
- async () => ReactNode
- function* () {}
- async function* () {}